ros/lib/utils.py

```diff
@@ -255,6 +267,18 @@ def system_allowed_in_ros(msg, reporter):
         is_ros = msg["input"]["platform_metadata"].get("is_ros")
         cloud_provider = msg["results"]["system"]["metadata"].get('cloud_provider')
     elif reporter == 'INVENTORY EVENTS':
-        is_ros = msg["platform_metadata"].get("is_ros")
         cloud_provider = msg['host']['system_profile'].get('cloud_provider')
+        if msg.get('type') == 'updated':
+            updated_event = True
+        else:
+            updated_event = False
+        if updated_event:
+            return is_valid_cloud_provider(cloud_provider)
```

Comment:
I didn't understand why `cloud_provider` check is ONLY applicable for updated event?